Python/compile.c

```diff
@@ -81,11 +81,24 @@ It's called a frame block to distinguish it from a basic block in the
 compiler IR.
 */
 
-enum fblocktype { LOOP, EXCEPT, FINALLY_TRY, FINALLY_END };
+enum fblocktype { LOOP, EXCEPT, FINALLY_TRY, FINALLY_END, ASYNC_WITH,
+                  HANDLER_CLEANUP, WITH, NO_TYPE };
+
```

Comment:
Nit:  Is there a reason WITH doesn't come before ASYNC_WITH?  Also, should HANDLER_CLEANUP be directly after the FINALLY_END?